locust/runners.py

```diff
@@ -938,6 +942,8 @@ def client_listener(self):
                 #    warnings.warn("The worker node's clock seem to be out of sync. For the statistics to be correct the different locust servers need to have synchronized clocks.")
             elif msg.type == "client_stopped":
                 client = self.clients[msg.node_id]
+                if client is None:
+                    continue
```

Comment:
Can you log a warning here?

Reply:
Of course, I will resubmit the pull request after modifying the code